src/gui/tray/CurrentAccountHeaderButton.qml

```diff
@@ -220,13 +220,15 @@ Button {
 
                 EnforcedPlainTextLabel {
                     id: emoji
+                    Layout.alignment: Qt.AlignLeft | Qt.AlignTop
                     visible: UserModel.currentUser && UserModel.currentUser.statusEmoji !== ""
                     width: Style.userStatusEmojiSize
                     text: UserModel.currentUser ? UserModel.currentUser.statusEmoji : ""
+                    topPadding: -Style.accountLabelsSpacing
```

Comment:
IMO negative padding is not a great solution, it's prone to break in unpredictable ways down the line

I think in the context of this change it makes more sense to apply a spacing of 0 to items we don't want to have any spacing, and then give the specific items that need some padding that padding independently. Another solution would be to have a child column or row or container that applies the spacing to the child items that need it

Reply:
(I know we already do this in the user line component, but my take is that if we are modifying this code we should fix it there too)